### DISCUSSION TITLE
this software is malware !!!

### DISCUSSION THREAD

**Discussion by emredusmez** [mkaring/ConfuserEx#267] [General]
Windows Defender detected ransomware in the .exe file I locked with this version.

**Comment by mkaring** [comment]
This is usually caused by the packer protection. The reason is that actual malware is using ConfuserEx some time.
Please read this pinned issue for more details: #64

**Comment by owersite** [comment]
@emredusmez People are using Cfex to protect their malwares, like rats or keyloggers, so anti viruses may detect it as a malware because it's stub is used for bad stuff sometimes.
